cfme/tests/automate/test_automate_manual.py

```diff
@@ -930,3 +930,127 @@ def test_git_refresh_with_renamed_yaml():
         1716443
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_upload_blank_line():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: negative
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create blank zip(test.zip) and yaml(test.yml) file
+            2. Navigate to Automation > Automate > Import/Export and upload test.zip file
+            3. Navigate to Automation > Automate > Customization > Import/Export and upload test.yml
+        expectedResults:
+            1.
+            2. Error message should be displayed
+            3. Error message should be displayed
+
+    Bugzilla:
+        1720611
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_initiate_retirement_automate_method():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create a writeable domain
+            2. Create a /System/Request/retire_automation_service instance and set meth5 to
+               retire_automation_service.
+            3. Create a /System/Request/retire_automation_service method.
+               Sample code:
+               ```
+               service = $evm.root['service']
+               $evm.log(:info, "create_retire_request for  service #{service}")
+               request = $evm.execute(:create_retire_request, service)
+               $evm.log(:info, "Create request for create_retire_request for request #{request}")
+               ```
+            4. In Automation/Automate/Simulation:
+               >> Set Request => retire_automation_service
+               >> Select Object Attribute Type => Service
+               >> Selection => Service to be retired
+               and click on Submit
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Service should be retired from Automate method.
+
+    Bugzilla:
+        1700524
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_git_refresh_with_rapid_updates():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Have a git backed domain that imported cleanly (if possibly incompletely)
+            2. Break the domain in Git, or notice a method isn't visible because its matching .yaml
```

Comment:
What do you mean by breaking the Git domain?

Reply:
Breaking the git domain means to add broken .yaml or do not add .yaml file which is important while importing domain.
Or we can have a domain which is blank without any namespaces etc

Reply:
Got it, could you add some explanation of that in the `testSteps`?